airqo-platform/AirQo-frontend
# [Website] Ft upgrade packages

#### Summary of Changes (What does this PR do?)

- Specifies node engines required to run the app (node 14+)
- Upgrades node packages

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Navigate through the app in deploy preview to see if any page breaks or errors returned

#### What are the relevant tickets?

- [WEB-363]



[WEB-363]: https://airqoteam.atlassian.net/browse/WEB-363?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
